google_drive/tests/integration/xml/image.xml

```diff
@@ -0,0 +1,3 @@
+<vertical_demo>
+    <google-document embed_code="&lt;img src='https://docs.google.com/drawings/d/1lmmxboBM5c_0WCTjhAxBdkpqQb3T8VSwtuG0TRR1ODQ/pub?w=960&amp;h=720'&gt;"/>
```

Comment:
Is this hash something else that we might move into a settings file or constant/property/helper?